src/components/Form/CrossPodWriteForm.jsx

```diff
@@ -42,18 +55,21 @@ const CrossPodWriteForm = () => {
     }
   };
 
-  // Custom useField hook for handling form inputs
-  const { clearValue: clearDescription, _type, ...description } = useField('textarea');
-
   const clearInputFields = (event) => {
     event.target.reset();
```

Comment:
As mentioned before since we're already moving everything to MUI and useState instead of event.target, event.target doesn't really have anything to reset from the input fields in Line 59. We can remove this.